### PR TITLE
Fix command line parameter def

### PR DESCRIPTION
**Pull request by Philbrick**
Change downsampling from float to int
Make recently added parameters start with lower case.

### CODE REVIEW

File: endToEnd/endToEndTest.sh [hunk 1]
```diff
@@ -53,11 +53,12 @@ dcm2pnm ./endToEnd/level-5-frames-0-1.dcm ./endToEnd/test4GeneratedImage.ppm
 diff ./endToEnd/test4GeneratedImage.ppm ./endToEnd/test4ExpectedImage.ppm
 
 #test - generate bilinear Downsamped DICOM and compare with expected image
-./build/wsi2dcm $fileName ./endToEnd/ --seriesDescription test5  --levels 6 --startOn 5 --tileHeight 100  --tileWidth 100 --compression raw --BilinearDownsampeling
+./build/wsi2dcm $fileName ./endToEnd/ --seriesDescription test5  --levels 6 --startOn 5 --tileHeight 100  --tileWidth 100 --compression raw --bilinearDownsampeling
```
Comment (danielbeaudreau) [inline endToEnd/endToEndTest.sh]: spelling mistake, should be "Downsampling"

**Review by danielbeaudreau — CHANGES_REQUESTED** [review]
Thanks!

**Comment by danielbeaudreau** [comment]
Ah thanks for the fix, but my comment was actually referring to the BilinearDownsampeling flag itself. =P

File: endToEnd/endToEndTest.sh (hunk 1 above)
Comment (Philbrick) [reply endToEnd/endToEndTest.sh]: fixed

Comment (danielbeaudreau) [reply endToEnd/endToEndTest.sh]: Ah thanks for the fix, but my comment was actually referring to the BilinearDownsampeling flag itself. =P

**Comment by danielbeaudreau** [comment]
Could you point out where the buffer overflow issue is?

**Pull request merged**